editor/editor_settings.h

```diff
@@ -62,6 +62,13 @@ class EditorSettings : public Resource {
 		NETWORK_ONLINE,
 	};
 
+	enum InitialScreen {
```

Comment:
What is this used for? Mixed it up with the one in `DisplayServer` below, is this one that was meant in those cases? In that case the value is off by one

Reply:
Ah yes whoops. Let me fix